src/sentry/static/sentry/app/views/organizationEventsV2/events.jsx

```diff
@@ -15,10 +18,15 @@ import {getQuery} from './utils';
 
 class Events extends AsyncComponent {
   static propTypes = {
+    router: PropTypes.object,
     organization: SentryTypes.Organization.isRequired,
     view: SentryTypes.EventView.isRequired,
   };
 
+  state = {
+    zoomed: false,
```

Comment:
Is this only to force a re-render?

Reply:
I was replicating the behavior in the current iteration of the events view. It also has a `handleZoom` method that sets `zoomed` to true implying the default should be false.